### ISSUE TITLE
Turfs after the first one on a tile are ignored

### ISSUE THREAD

**Issue by DamianX** [SpaiR/StrongDMM#37]
![image](https://user-images.githubusercontent.com/6307265/81894376-e3176c80-95af-11ea-99ce-e4406e90590d.png)

![image](https://user-images.githubusercontent.com/6307265/81894514-3ee1f580-95b0-11ea-889f-06cc86fd1110.png)

DM supports having more than one /turf on a tile.
If I recall correctly, turfs after the first are added as images to the first one. Or it could be the other way around. I don't remember.
/vg/station uses this feature to make shuttle corners with space underneath them, but we're probably stupid.

Is this something you would want to support?

**Comment by SpaiR** [comment]
Technically speaking, BYOND doesn't support multiple turfs per tile.

http://www.byond.com/docs/ref/#/turf
> When you create a new turf with new(), it always replaces the old one. 

DM map editor do has an undocumented behavior when it allows multiple turfs. In game they are merged into one. But I found this unintuitive and inconsistent with the documentation. So I've made rule 'one turf per tile' on purpose.

Other codebases (TG and all downstreams) have their shuttle walls implemented as objects, not turfs. I would recommend to do the same.
